.gitignore

```diff
@@ -74,6 +74,7 @@ target/
 
 # Jupyter Notebook
 .ipynb_checkpoints
+notebooks/
```

Comment:
You might want to add notebooks to the ignore list for flakeheaven as well (unless you want the notebooks linted)